File: CS/Helpers/DragDrop/IDropGridColumnHeaderSupport.cs
using DevExpress.XtraGrid.Columns;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Windows.Forms;

namespace dxQ529392 {
    public interface IDropGridColumnHeaderSupport {
        void ProcessDropObject(string columnFieldName, Point location, Control control);
        bool SupportsControl(Control control);
    }
}

File: VB/Helpers/DragDrop/IDropGridColumnHeaderSupport.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraGrid.Columns
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Windows.Forms

Namespace dxQ529392
	Public Interface IDropGridColumnHeaderSupport
		Sub ProcessDropObject(ByVal columnFieldName As String, ByVal location As Point, ByVal control As Control)
		Function SupportsControl(ByVal control As Control) As Boolean
	End Interface
End Namespace
